### PR TITLE
responsive mobile

### PR DESCRIPTION
**Pull request by maxmodlol**

### CODE REVIEW

File: css/csshome.css [hunk 1]
```diff
@@ -12,10 +11,628 @@
...
+.sliderdiv{
+  box-shadow: 0px 0px 1px 1px  rgba(0,0,0,.2);
+  margin: 10px!important;
+  padding: 10px;
+}
+.sliderdiv img{
+  width: 100%;
+}
+.sliderdiv p{
+  width: 100%;
```
Comment (Abdallah-Via) [inline css/csshome.css:63]: The component is already "div" there is no need to mention it in the class name
please change it in all attributes

File: css/csshome.css [hunk 2]
```diff
@@ -12,10 +11,628 @@
...
+  display: none;
+}
+.hammemnu{
+  display: none;
+}
+.mobile-container{
+  display: none;
+}
+.responsivenav{
+  display: none;
```
Comment (Abdallah-Via) [inline css/csshome.css:24]: use bem naming convention

File: css/csshome.css [hunk 3]
```diff
@@ -12,10 +11,628 @@
...
+}
+.picssecmain{
+  height: 450px;
+}
+
+@media only screen and (max-width: 768px) {
+  *{ font-size: 1rem;
+    font-weight: 300;
+
 }
```
Comment (Abdallah-Via) [inline css/csshome.css:81]: use @media at the end of css file

File: css/csshome.css [hunk 4]
```diff
@@ -12,10 +11,628 @@
...
+  .smallsec a{
+    font-family: Arial !important;
+    font-size: 16px!important;
+    font-weight: bold!important;
+
+  }
+  .smallsec p{
+    display: none;
+  }
+  .smallsec div:last-child{
```
Comment (Abdallah-Via) [inline css/csshome.css:119]: there is no need to use :last-child with div, think in another way to design your code

File: css/csshome.css [hunk 5]
```diff
@@ -12,10 +11,628 @@
...
+
+
+  }
+  .footer .subfot   img{
+
+
+    max-width: 100px;
+
+
+  }
```
Comment (Abdallah-Via) [inline css/csshome.css:238]: make sure the css attributes (in the whole file) has no spaces between {}

File: css/csshome.css [hunk 6]
```diff
@@ -12,10 +11,628 @@
...
+  .footer .subfot aside{
+    max-width: 5%;
+  float: left;
+    margin-left: 30vh !important;
+    margin-top: 250px !important;
+
+
+
+  }
+  .footer .subfot  div i{
```
Comment (Abdallah-Via) [inline css/csshome.css:221]: there is no need to make css classes very deep, if you want to style `i` create a style for it directly (check BEM guide)

File: css/csshome.css [hunk 7]
```diff
@@ -12,10 +11,628 @@
...
+    color: #444;
+    cursor: pointer;
+    padding: 18px;
+    width: 100%;
+    text-align: left;
+    border: none;
+    outline: none;
+    transition: 0.4s;
+  }
+  .footeraccordion{
```
Comment (Abdallah-Via) [inline css/csshome.css:262]: make sure the names are more clear

File: css/csshome.css [hunk 8]
```diff
@@ -12,10 +11,628 @@
...
+    display: block !important;
```
Comment (Abdallah-Via) [inline css/csshome.css:263]: I don't want any !important; in the css file

File: css/csshome.css [hunk 9]
```diff
@@ -12,10 +11,628 @@
...
+  }
+  .icon{display:block;
+    cursor: pointer;
+
+    margin-right:30vh ;
+  }
+  .logo{
+    margin-left: -40px;
+  }
+  .nav:hover .navitem {
```
Comment (Abdallah-Via) [inline css/csshome.css:340]: don't create an empty blocks

File: css/csshome.css [hunk 10]
```diff
@@ -12,10 +11,628 @@
...
+      max-width: 400px;
+      align-items: stretch;
+    }
+    .footerend i {
+      font-size: 16px;color: gray;font-family: Roboto;
+    }
+    .footerend div{
+  align-content: center;
+      background-color:#eee;
+    }
```
Comment (Abdallah-Via) [inline css/csshome.css:388]: make sure the css blocks are we indented

File: css/csshome.css [hunk 11]
```diff
@@ -12,10 +11,628 @@
...
+
+}
+.divcontentmaindiv{
+  float: right;margin-top: -160px;margin-left: 320px;
+}
+@media (min-width: 640px) { body {font-size:1rem;} }
+@media (min-width:960px) { body {font-size:1.2rem;} }
+@media (min-width:1100px) { body {font-size:1.2rem;} }
+@media only screen and (max-width: 760px){
+
```
Comment (Abdallah-Via) [inline css/csshome.css:498]: there is no need to create this amount of @media sections

File: css/csshome.css [hunk 12]
```diff
@@ -12,10 +11,628 @@
...
+  }
+  .divcontentmaindiv h2{
+    font-size: 14px!important;
+  }
+  .divcontentmaindiv p{
+    display: none;
+    font-size: 12px !important;
+  }
+
+  .divrightcontentmain div:first-child{
```
Comment (Abdallah-Via) [inline css/csshome.css:526]: - don't use div in class names 
- don't use div:first-child, if you want the first child just use .classname:first-child

File: css/csshome.css [hunk 13]
```diff
@@ -12,10 +11,628 @@
...
+  }
+  .headercontent p {
+    max-width: 350px;
+  }
+  .header .divrightcontent {
+    display: inline-block;
+  }
+
+}
+body{
```
Comment (Abdallah-Via) [inline css/csshome.css:635]: it is preferred that body block be at the first line of css file

File: index.html [hunk 14]
```diff
@@ -20,37 +20,90 @@
...
+  }
+  function myFunction() {
+    var x = document.getElementById("myLinks");
+    if (x.style.display === "block") {
+      x.style.display = "none";
+    } else {
+      x.style.display = "block";
+    }
+  }
+</script>
```
Comment (Abdallah-Via) [inline index.html:35]: the script tag should be before `</body> `

File: index.html [hunk 15]
```diff
@@ -20,37 +20,90 @@
...
+
+<body style="">
+<div class="container">
+<div class="nav" id="nav">
+  <div class="logo" onclick="">
   <a href=""  style="padding-left: 50px;padding-right: 10px">
```
Comment (Abdallah-Via) [inline index.html:41]: make sure to use a good indentation in the whole file

File: index.html [hunk 16]
```diff
@@ -20,37 +20,90 @@
...
+</div>
+</div>
+<div class="header">
+    <div class="headercontent">
   <div style="width: 500px;margin-left: 60px;margin-top:60px ;box-shadow: 0 1px 2px 0 rgba(0,0,0,.2);">
     <img src="img/tank.jpg" style="object-fit:fill;width:500px">
-    <div class="square" style=";width: 100px;height: 10px;margin-left: 20px;margin-top: -5px;"></div>
   </div>
-  <div style="float: left;width: 400px">
+      <div class="square" style=";width: 100px;height: 10px;margin-left: 60px;margin-top: -5px;"></div>
```
Comment (Abdallah-Via) [inline index.html:99]: remove inline style

File: index.html [hunk 17]
```diff
@@ -342,12 +405,12 @@ <h2 style=" white-space: break-spaces"><a href="">`Shared resentments ' but Russ
   </div>
 <div style="display: flex">
```
Comment (Abdallah-Via) [inline index.html:406]: remove inline style to all file

File: css/csshome.css [hunk 18]
```diff
@@ -1,7 +1,6 @@
 .nav{
   border: 1px solid beige;
-
-  width: 100%;
+  width: 1400px;
   height:60px;
   box-shadow: 0 1px 2px 0 rgba(0,0,0,.2);
   overflow: hidden;
```
Comment (Abdallah-Via) [inline css/csshome.css:6]: test

File: css/csshome.css [hunk 19]
```diff
@@ -1,7 +1,6 @@
 .nav{
   border: 1px solid beige;
-
-  width: 100%;
+  width: 1400px;
   height:60px;
```
Comment (AbdallahSabri) [inline css/csshome.css:4]: test

**Pull request closed without merging**